source/user-manual/reference/centralized-configuration.rst

```diff
@@ -120,26 +120,52 @@ Options
 +-------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | **name**    | Allows assignment of the block to one particular agent.                                                                                                           |
 +             +-------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
-|             | Allowed values                                        | Any agent name                                                                                            |
+|             | Allowed values                                        | Any regular expression that matches the agent name .                                                      |
 +-------------+-------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
 | **os**      | Allows assignment of the block to an operating system.                                                                                                            |
 +             +-------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
-|             | Allowed values                                        | Any OS family                                                                                             |
+|             | Allowed values                                        | Any regular expression that matches the agent OS information.                                             |
 +-------------+-------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
 | **profile** | Allows assignment of a profile name to a block. Any agent configured to use the defined :ref:`profile <reference_ossec_client_config_profile>` may use the block. |
 +             +-------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
-|             | Allowed values                                        | Any defined profile                                                                                       |
+|             | Allowed values                                        | Any regular expression that matches the defined profile.                                                  |
 +-------------+-------------------------------------------------------+-----------------------------------------------------------------------------------------------------------+
 
-Examples
+.. topic:: Example
```

Comment:
We could have a more varied display of regex possibilities. For example `^agent(11|12|13)`

Reply:
The regular expression used for this is very limited, I used the one that worked. 
![image](https://github.com/wazuh/wazuh-documentation/assets/13010397/74bf3a1e-ad55-41ba-92f9-ba34debddf23)

![image](https://github.com/wazuh/wazuh-documentation/assets/13010397/b108fece-b9e7-483a-98a2-7c24e6da16b2)

![image](https://github.com/wazuh/wazuh-documentation/assets/13010397/ddf4c883-fdfe-48ce-ba4a-abde2839bfc6)